README.md

```diff
@@ -1342,6 +1342,220 @@ Let's see how many code lines are necessary to encrypt and search some data usin
 
 These are all the code changes! 🎉
 
+
+# Example 12. AcraTranslator usage
+
+Learn more details
+about [AcraTranslator](https://docs.cossacklabs.com/acra/acra-in-depth/architecture/acratranslator/#acratranslator-an-api-service)
+architecture.
+
+## 1. Installation
+
+```bash
+$ curl https://raw.githubusercontent.com/cossacklabs/acra-engineering-demo/master/run.sh | \
+    bash -s -- acra-translator
+```
+
+This command downloads a simple Go API Server application that stores the data in a database, AcraTranslator Docker
+containers, MongoDB
+database, sets up the environment, configures Go application to connect to AcraTranslator and MongoDB, and provides an
+API(Swagger) for you to try.
```

Comment:
can we generate postman config for that? a lot of people relies on that and looks very useful